src/Angular.js

```diff
@@ -705,12 +707,20 @@ function copy(source, destination, stackSource, stackDest) {
     throw ngMinErr('cpws',
       "Can't copy! Making copies of Window or Scope instances is not supported.");
   }
+  if (isFloat32Array(destination)) {
+    throw ngMinErr('cpfad', "Can't copy! Float32Array destination cannot be mutated.");
+  }
 
   if (!destination) {
     destination = source;
     if (source) {
       if (isArray(source)) {
         destination = copy(source, [], stackSource, stackDest);
+      } else if (isFloat32Array(source)) {
+        destination = new Float32Array(source.length);
+        for (var j = 0; j < source.length; j++) {
+          destination[j] = source[j];
```

Comment:
this is not actually necessary, you can just do `destination = new source.constructor(source)` (or `new <specific constructor>(source)` if worried about `constructor` being overwritten)